backend/package.json

```diff
@@ -25,6 +27,7 @@
     "dotenv": "^8.2.0",
     "express": "^4.19.2",
     "express-rate-limit": "^6.2.0",
+    "firebase": "^10.12.3",
     "firebase-admin": "^9.5.0",
```

Comment:
(see comment below)
If the 3 firebase packages you installed aren't actively being used anymore, could we uninstall them?